ztag/schema.py

```diff
@@ -532,6 +532,31 @@ def ztag_zgrab2_transformed(service, results):
                              "the handshake.")
 }))
 
+ztag_mongodb = ztag_zgrab2_transformed(service="MongoDB", results=SubRecord({
+        "build_info": SubRecord({
+            "version": String(doc="Version of mongodb server"),
+            "git_version": String(doc="Git Version of mongodb server"),
+            "max_wire_version": Signed32BitInteger(),
+            "build_environment": SubRecord({
+                "dist_mod": String(),
+                "dist_arch": String(),
+                "cc": String(),
+                "cc_flags": String(),
+                "cxx": String(),
+                "cxx_flags": String(),
+                "link_flags": String(),
+                "target_arch": String(),
+                "target_os": String()})}),
```

Comment:
**Nit:** I'd put each of the `})` on their own line.

Reply:
OK